modules/op-release-notes-1-1.adoc

```diff
@@ -17,6 +17,8 @@
 In addition to the fixes and stability improvements, here is a highlight of what’s new in OpenShift Pipelines 1.1.
 
 === Pipelines
+
+* Support for using Workspaces instead of PipelineResources is now provided. We recommend using Workspaces in OpenShift Pipelines, as PipelineResources are opaque, difficult to debug, limited in scope, and make Tasks less reusable. For more details on Workspaces, see the detailed OpenShift Pipelines concepts section.
```

Comment:
A suggested revision for readability and style:

> Workspaces can now be used instead of PipelineResources. Use of Workspaces is recommended in OpenShift Pipelines, as PipelineResources are difficult to debug, limited in scope, and can make Tasks less reusable. For more details on Workspaces, see **Understanding OpenShift Pipelines** (with link to assembly).

(For the last part, I don't think the reader will be able to understand what is meant by "concepts section.")

Reply:
I would have loved to add a link but xrefs can not be added to modules, right?

Reply:
Oh, right. Sorry, didn't realize this was a module so ignore my link suggestion.